2022-05-12 22:25:21,269 - INFO - Try no control scenario...
2022-05-12 22:25:28,199 - INFO - [º∆À„≥ˆ¥Ì¡À] x = 1 / 0
Traceback (most recent call last):
  File "D:/Learn/ZCPaper/Collaborative Freight Transport/Model1/test.py", line 265, in <module>
    x = 1 / 0
ZeroDivisionError: division by zero
2022-05-12 22:26:03,023 - INFO - Try no control scenario...
2022-05-12 22:26:05,754 - INFO - No control scenario complete!
2022-05-12 22:26:06,516 - INFO - Finally...
2022-05-12 22:26:07,323 - INFO - Execution time of no control: 3493.2977
2022-05-12 22:26:07,866 - INFO - End
2022-05-12 22:26:08,170 - INFO - Try holding control scenario...
2022-05-12 22:33:09,742 - INFO - Try no control scenario...
2022-05-12 22:33:12,543 - INFO - No control scenario complete!
2022-05-12 22:33:13,281 - INFO - Finally...
2022-05-12 22:33:13,839 - INFO - Execution time of no control: 3539.2516
2022-05-12 22:33:14,419 - INFO - End
2022-05-12 22:33:15,422 - INFO - Try holding control scenario...
2022-05-12 23:16:02,773 - INFO - Holding control scenario complete!
2022-05-12 23:16:03,596 - INFO - Finally...
2022-05-12 23:16:03,990 - INFO - Execution time of holding control: 2568174.5261999997
2022-05-12 23:16:04,677 - INFO - End
2022-05-12 23:16:05,175 - INFO - Try FTNC scenario...
2022-05-12 23:37:41,380 - INFO - FTNC scenario complete!
2022-05-12 23:37:42,041 - INFO - Finally...
2022-05-12 23:37:42,805 - INFO - Execution time of FTNC: 1296866.268
2022-05-12 23:37:43,168 - INFO - End
2022-05-12 23:37:44,041 - INFO - Try average FTNC scenario, when demand is 5
2022-05-13 00:59:03,533 - INFO - Average FTNC scenario complete, when demand is 5
2022-05-13 00:59:04,606 - INFO - Finally...
2022-05-13 00:59:05,032 - INFO - Execution time of average FTNC when demand is 5 is 4880564.6603999995
2022-05-13 00:59:05,705 - INFO - End
2022-05-13 00:59:06,409 - INFO - Try average FTNC scenario, when demand is 10
2022-05-13 02:50:07,957 - INFO - Average FTNC scenario complete, when demand is 10
2022-05-13 02:50:08,832 - INFO - Finally...
2022-05-13 02:50:09,344 - INFO - Execution time of average FTNC when demand is 10 is 11544790.5357
2022-05-13 02:50:09,759 - INFO - End
2022-05-13 02:50:10,507 - INFO - Try average FTNC scenario, when demand is 15
2022-05-14 17:14:25,347 - INFO - Average FTNC scenario complete, when demand is 15
2022-05-14 17:14:26,164 - INFO - Finally...
2022-05-14 17:14:27,197 - INFO - Execution time of average FTNC when demand is 15 is 149800112.5142
2022-05-14 17:14:27,986 - INFO - End
2022-05-14 17:14:28,961 - INFO - Try average FTNC scenario, when demand is 20
2022-05-18 14:12:48,975 - INFO - Try no control scenario...
2022-05-18 14:12:52,593 - INFO - No control scenario complete!
2022-05-18 14:12:53,327 - INFO - Finally...
2022-05-18 14:12:54,025 - INFO - Execution time of no control: 4352.2600999999995
2022-05-18 14:12:54,874 - INFO - End
2022-05-18 14:12:55,798 - INFO - Try holding control scenario...
2022-05-18 14:15:50,966 - INFO - Try no control scenario...
2022-05-18 14:15:53,907 - INFO - No control scenario complete!
2022-05-18 14:15:54,909 - INFO - Finally...
2022-05-18 14:15:55,913 - INFO - Execution time of no control: 3942.1055
2022-05-18 14:15:56,335 - INFO - End
2022-05-18 14:15:57,097 - INFO - Try holding control scenario...
2022-05-18 15:16:01,964 - INFO - Holding control scenario complete!
2022-05-18 15:16:02,795 - INFO - Finally...
2022-05-18 15:16:03,605 - INFO - Execution time of holding control: 3605697.8696
2022-05-18 15:16:04,040 - INFO - End
2022-05-18 15:16:04,792 - INFO - Try FTNC scenario...
2022-05-18 15:17:22,387 - DEBUG - Except: [Errno 13] Permission denied: './Log/ProcessLog.txt'
2022-05-18 15:17:23,079 - DEBUG - Traceback (most recent call last):
  File "D:/Learn/ZCPaper/Collaborative Freight Transport/Model1/main.py", line 137, in <module>
    FTNC.Analysis()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 674, in Analysis
    self.dynamic_programming()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 600, in dynamic_programming
    with open('./Log/ProcessLog.txt','a') as file:
PermissionError: [Errno 13] Permission denied: './Log/ProcessLog.txt'

2022-05-18 15:17:23,715 - INFO - Finally...
2022-05-18 15:17:24,349 - INFO - Execution time of FTNC: 78924.35579999999
2022-05-18 15:17:25,276 - INFO - End
2022-05-18 15:17:26,199 - INFO - Try average FTNC scenario, when demand is 5
2022-05-18 15:17:35,718 - INFO - Except: [Errno 13] Permission denied: './Log/ProcessLog.txt'
2022-05-18 15:17:36,709 - DEBUG - Traceback (most recent call last):
  File "D:/Learn/ZCPaper/Collaborative Freight Transport/Model1/main.py", line 169, in cal_exec_time
    object.Average_Analysis()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in Average_Analysis
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in <listcomp>
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 904, in item_process
    self.dynamic_programming()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 600, in dynamic_programming
    with open('./Log/ProcessLog.txt','a') as file:
PermissionError: [Errno 13] Permission denied: './Log/ProcessLog.txt'

2022-05-18 15:17:37,486 - INFO - Finally...
2022-05-18 15:17:38,116 - INFO - Execution time of average FTNC when demand is 5 is 11286.5055
2022-05-18 15:17:38,793 - INFO - End
2022-05-18 15:17:39,446 - INFO - Try average FTNC scenario, when demand is 10
2022-05-18 15:17:42,376 - INFO - Except: [Errno 13] Permission denied: './Log/ProcessLog.txt'
2022-05-18 15:17:43,321 - DEBUG - Traceback (most recent call last):
  File "D:/Learn/ZCPaper/Collaborative Freight Transport/Model1/main.py", line 169, in cal_exec_time
    object.Average_Analysis()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in Average_Analysis
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in <listcomp>
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 904, in item_process
    self.dynamic_programming()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 600, in dynamic_programming
    with open('./Log/ProcessLog.txt','a') as file:
PermissionError: [Errno 13] Permission denied: './Log/ProcessLog.txt'

2022-05-18 15:17:43,932 - INFO - Finally...
2022-05-18 15:17:44,307 - INFO - Execution time of average FTNC when demand is 10 is 17733.4614
2022-05-18 15:17:44,768 - INFO - End
2022-05-18 15:17:45,492 - INFO - Try average FTNC scenario, when demand is 15
2022-05-18 15:17:50,143 - INFO - Except: [Errno 13] Permission denied: './Log/ProcessLog.txt'
2022-05-18 15:17:50,822 - DEBUG - Traceback (most recent call last):
  File "D:/Learn/ZCPaper/Collaborative Freight Transport/Model1/main.py", line 169, in cal_exec_time
    object.Average_Analysis()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in Average_Analysis
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in <listcomp>
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 904, in item_process
    self.dynamic_programming()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 600, in dynamic_programming
    with open('./Log/ProcessLog.txt','a') as file:
PermissionError: [Errno 13] Permission denied: './Log/ProcessLog.txt'

2022-05-18 15:17:51,950 - INFO - Finally...
2022-05-18 15:17:52,630 - INFO - Execution time of average FTNC when demand is 15 is 25751.1354
2022-05-18 15:17:52,956 - INFO - End
2022-05-18 15:17:53,449 - INFO - Try average FTNC scenario, when demand is 20
2022-05-18 15:29:39,456 - INFO - Try FTNC scenario...
2022-05-18 15:38:09,386 - INFO - Try no control scenario...
2022-05-18 15:38:11,946 - INFO - No control scenario complete!
2022-05-18 15:38:12,427 - INFO - Finally...
2022-05-18 15:38:13,028 - INFO - Execution time of no control: 3040.8073
2022-05-18 15:38:13,600 - INFO - End
2022-05-18 15:38:14,494 - INFO - Try holding control scenario...
2022-05-18 16:38:18,022 - INFO - Holding control scenario complete!
2022-05-18 16:38:19,364 - INFO - Finally...
2022-05-18 16:38:19,735 - INFO - Execution time of holding control: 3604869.4834000003
2022-05-18 16:38:20,380 - INFO - End
2022-05-18 16:38:21,786 - INFO - Try FTNC scenario...
2022-05-18 16:38:51,949 - DEBUG - Except: [Errno 13] Permission denied: './Log/ProcessLog.txt'
2022-05-18 16:38:52,354 - DEBUG - Traceback (most recent call last):
  File "D:/Learn/ZCPaper/Collaborative Freight Transport/Model1/main.py", line 137, in <module>
    FTNC.Analysis()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 674, in Analysis
    self.dynamic_programming()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 600, in dynamic_programming
    with open('./Log/ProcessLog.txt','a') as file:
PermissionError: [Errno 13] Permission denied: './Log/ProcessLog.txt'

2022-05-18 16:38:52,934 - INFO - Finally...
2022-05-18 16:38:53,743 - INFO - Execution time of FTNC: 31149.367899999997
2022-05-18 16:38:54,882 - INFO - End
2022-05-18 16:38:55,597 - INFO - Try average FTNC scenario, when demand is 5
2022-05-18 16:38:58,352 - INFO - Except: [Errno 13] Permission denied: './Log/ProcessLog.txt'
2022-05-18 16:38:59,231 - DEBUG - Traceback (most recent call last):
  File "D:/Learn/ZCPaper/Collaborative Freight Transport/Model1/main.py", line 169, in cal_exec_time
    object.Average_Analysis()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in Average_Analysis
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in <listcomp>
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 904, in item_process
    self.dynamic_programming()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 600, in dynamic_programming
    with open('./Log/ProcessLog.txt','a') as file:
PermissionError: [Errno 13] Permission denied: './Log/ProcessLog.txt'

2022-05-18 16:39:00,160 - INFO - Finally...
2022-05-18 16:39:00,568 - INFO - Execution time of average FTNC when demand is 5 is 4564.428900000001
2022-05-18 16:39:01,293 - INFO - End
2022-05-18 16:39:02,195 - INFO - Try average FTNC scenario, when demand is 10
2022-05-18 16:39:04,747 - INFO - Except: [Errno 13] Permission denied: './Log/ProcessLog.txt'
2022-05-18 16:39:05,529 - DEBUG - Traceback (most recent call last):
  File "D:/Learn/ZCPaper/Collaborative Freight Transport/Model1/main.py", line 169, in cal_exec_time
    object.Average_Analysis()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in Average_Analysis
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in <listcomp>
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 904, in item_process
    self.dynamic_programming()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 600, in dynamic_programming
    with open('./Log/ProcessLog.txt','a') as file:
PermissionError: [Errno 13] Permission denied: './Log/ProcessLog.txt'

2022-05-18 16:39:06,087 - INFO - Finally...
2022-05-18 16:39:06,470 - INFO - Execution time of average FTNC when demand is 10 is 10491.2795
2022-05-18 16:39:07,068 - INFO - End
2022-05-18 16:39:07,867 - INFO - Try average FTNC scenario, when demand is 15
2022-05-18 16:39:11,370 - INFO - Except: [Errno 13] Permission denied: './Log/ProcessLog.txt'
2022-05-18 16:39:11,883 - DEBUG - Traceback (most recent call last):
  File "D:/Learn/ZCPaper/Collaborative Freight Transport/Model1/main.py", line 169, in cal_exec_time
    object.Average_Analysis()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in Average_Analysis
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in <listcomp>
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 904, in item_process
    self.dynamic_programming()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 600, in dynamic_programming
    with open('./Log/ProcessLog.txt','a') as file:
PermissionError: [Errno 13] Permission denied: './Log/ProcessLog.txt'

2022-05-18 16:39:13,384 - INFO - Finally...
2022-05-18 16:39:14,318 - INFO - Execution time of average FTNC when demand is 15 is 17787.435100000002
2022-05-18 16:39:15,169 - INFO - End
2022-05-18 16:39:15,685 - INFO - Try average FTNC scenario, when demand is 20
2022-05-18 16:40:29,100 - INFO - Except: [Errno 13] Permission denied: './Log/ProcessLog.txt'
2022-05-18 16:40:30,320 - DEBUG - Traceback (most recent call last):
  File "D:/Learn/ZCPaper/Collaborative Freight Transport/Model1/main.py", line 169, in cal_exec_time
    object.Average_Analysis()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in Average_Analysis
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 907, in <listcomp>
    final_data_result=[item_process() for i in range(repeat_time)]
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 904, in item_process
    self.dynamic_programming()
  File "D:\Learn\ZCPaper\Collaborative Freight Transport\Model1\CollaborativeScenario.py", line 600, in dynamic_programming
    with open('./Log/ProcessLog.txt','a') as file:
PermissionError: [Errno 13] Permission denied: './Log/ProcessLog.txt'

2022-05-18 16:40:31,615 - INFO - Finally...
2022-05-18 16:40:32,876 - INFO - Execution time of average FTNC when demand is 20 is 96018.685
2022-05-18 16:40:34,119 - INFO - End
2022-05-18 16:40:35,776 - INFO - Execution time of average FTNC is [4564.428900000001, 5926.8506, 7296.155600000002, 78231.2499]
2022-05-18 16:40:36,922 - INFO - Total execution time of average FTNC is 96018.685
2022-05-18 16:40:37,813 - INFO - Try FTHC scenario...
2022-05-18 17:42:31,103 - INFO - Try FTNC scenario...
